apps/emqx_auth_jwt/src/emqx_auth_jwt_svr.erl

```diff
@@ -114,7 +115,8 @@ handle_cast(_Msg, State) ->
 
 handle_info({timeout, _TRef, refresh}, State = #state{addr = Addr}) ->
     NState = try
-                 true = ets:insert(?TAB, {remote, request_jwks(Addr)})
+                 true = ets:insert(?TAB, {remote, request_jwks(Addr)}),
```

Comment:
I found 4.3 also has this problem. Is it better to fix on 4.3 first and then sync to 4.4?

Reply:
https://github.com/emqx/emqx/pull/8337